src/utils/index.ts

```diff
@@ -43,14 +67,41 @@ export const downloadTheme = async (
 ) => {
   const theme: DefaultThemeType = { ...defaultTheme, colors: colorMapping };
 
-  const allColors = { ...colorMapping, ...customColorMapping };
+  const allColors: Record<string, string> = {
+    ...colorMapping,
+    ...customColorMapping,
+  };
+  const colors = getPalette(allColors, luminanceSteps);
```

Comment:
We should move the line 74-91 to a separate function.
Maybe we can even create an own file under `outputs` directory just for sketch